src/helpers/helpers.easing.js

```diff
@@ -1,194 +1,84 @@
 import {PI, TAU, HALF_PI} from './helpers.math';
 
+const atEdge = (t) => t === 0 || t === 1;
+const elasticIn = (t, s, p) => -(Math.pow(2, 10 * (t -= 1)) * Math.sin((t - s) * TAU / p));
+const elasticOut = (t, s, p) => Math.pow(2, -10 * t) * Math.sin((t - s) * TAU / p) + 1;
+// Math.pow(2, -10 * t) * Math.sin((t - 0.075) * TAU / 0.3) + 1,
 /**
  * Easing functions adapted from Robert Penner's easing equations.
  * @namespace Chart.helpers.easing.effects
  * @see http://www.robertpenner.com/easing/
  */
 const effects = {
-  linear(t) {
-    return t;
-  },
+  linear: t => t,
 
-  easeInQuad(t) {
-    return t * t;
-  },
+  easeInQuad: t => t * t,
 
-  easeOutQuad(t) {
-    return -t * (t - 2);
-  },
+  easeOutQuad: t => -t * (t - 2),
 
-  easeInOutQuad(t) {
-    if ((t /= 0.5) < 1) {
-      return 0.5 * t * t;
-    }
-    return -0.5 * ((--t) * (t - 2) - 1);
-  },
+  easeInOutQuad: t => ((t /= 0.5) < 1)
```

Comment:
probably don't need the parens around `(t /= 0.5) < 1`.  to me the extra parens make it less readable because there's more to parse. there's a handful of others like this as well

Reply:
its actually needed, because its an assignment. Is really bad practice, but did not bother trying to change those.